login.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="ko">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>판다마켓-로그인</title>
+        <link rel="stylesheet" href="style2.css" />
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    <body>
+        <main class="container">
+            <div class="logo-wrap">
+                <a href="/"> <img src="images/logo.png" alt="판다마켓 로고" /></a>
+            </div>
+            <form method="post">
+                <div class="form-group">
+                    <label for="email">이메일</label>
+                    <div><input type="email" id="email" name="email" placeholder="이메일을 입력해주세요" /></div>
+                </div>
+                <div class="form-group">
+                    <label for="password">비밀번호</label>
+                    <div class="password-wrap">
+                        <input type="password" id="password" name="password" placeholder="비밀번호를 입력해 주세요" />
+                        <!--<button class="password-button" type="button"><img href="images/unvisible_eye.png" /></button>-->
+                        <!--눈아이콘을 만들고 싶었는데 작동하지 않아요-->
```

Comment:
💊 제안
제가 봤을 때 아래처럼 수정하고 CSS를 변경해주시면 button이 잘 보이실거에요~
이걸 기반으로 디자인과 동일하게 구현해보세요!

 +버튼 클릭 시 아이콘이 변경되는 것은 추후 구현할 예정이니 우선은 디자인대로 구현만 해보세요!

```suggestion
                        <button class="password-button" type="button"><img src="images/unvisible_eye.png" /></button>
```
```css
.password-button {
  position: absolute;
  right: 24px;
  width: 24px;
  height: 24px;
}

.password-button img {
  width: 100%;
}
```